packages/flutter/lib/src/widgets/scroll_position_with_single_context.dart

```diff
@@ -84,11 +84,6 @@ class ScrollPositionWithSingleContext extends ScrollPosition implements ScrollAc
     return super.setPixels(newPixels);
   }
 
-  @override
-  void correctBy(double correction) {
```

Comment:
I left the correctPixels in the super alone like we talked about but this is an actual bug since it circumvents its super.correctBy and doesn't actually do anything different.

Reply:
cool